Elastica/EventSubscriber/UpdateContentIndexedSubscriber.php

```diff
@@ -33,14 +34,12 @@ public function updateIndexedContent(ContentEvent $event)
     {
         $content = $event->getContent();
 
-        if (!$content->getStatus()->isPublished()) {
-            return;
-        }
-
         $lastPublishedContent = $this->contentRepository->findLastPublishedVersion($content->getContentId(), $content->getLanguage());
 
-        if ($content->getVersion() >= $lastPublishedContent->getVersion()) {
-            $this->contentIndexor->index($content);
+        if ($lastPublishedContent instanceof ContentInterface) {
```

Comment:
Yes but, if you change a content in other status that published like pending, the you index the lastPublishedContent whereas already indexed